controllers/activegate/capability/capability.go

```diff
@@ -126,14 +132,52 @@ func NewKubeMonCapability(crProperties *dynatracev1alpha1.CapabilityProperties)
 				Name: trustStoreVolume,
 				VolumeSource: v1.VolumeSource{
 					EmptyDir: &v1.EmptyDirVolumeSource{},
-				},
-			}},
+				}},
+			},
 		},
 	}
+
+	setTlsConfig(agSpec, &c.capabilityBase)
+
+	return c
+}
+
+func setTlsConfig(agSpec *dynatracev1alpha1.ActiveGateSpec, c *capabilityBase) {
```

Comment:
would be nicer as a method with a pointer receiver. E.g.:
`func (c *capabilityBase) setTlsConfig(agSpec)`